localisation/00_zuwc_united_localisation.csv

```diff
@@ -223,38 +223,38 @@ EVTDESC_WCWAR_312;[Root.PrimaryTitle.GetFOA], I'm giving you [FromFromFrom.GetNa
 EVTOPTC_WCBDP_65_tooltip;You reject the union.\n;;Ihr lehnt das B�ndnis ab.\n;;;;;;;;;;;x
 EVTDESC_WCBDP_67_alliance_case_01;The leader of the Laughing Skull gladly agreed to help us. Although their style of warfare is truly frightening, even with low numbers the clan is stronger and more dangerous than many other green-skinned. We should be glad that these madmen are on our side...;;Der Anf�hrer der lachenden Sch�del ist gerne bereit, uns zu helfen. Aufgrund ihres wahrhaft be�ngstigenden Kampfstils war dieser Clan trotz seiner geringen Anzahl m�chtiger und gef�hrlicher als viele andere Gr�nh�ute. Wir sollten froh sein, dass diese Verr�ckten auf unserer Seite sind...;;;;;;;;;;;x
 EVTDESC_WCBDP_67_alliance_case_02;The leader of the Laughing Skull gladly agreed to help us. Our soldiers are frightened by their laughter and bloodthirstiness, so it was best to leave the orcs at the Dark Portal, promising them land in the future. They will be able to intercept orcish clans remaining at or near the Dark Portal.;;Der Anf�hrer der lachenden Sch�del ist gerne bereit, uns zu helfen. Unsere Soldaten werden noch immer von ihrem Gel�chter und ihrer Blutr�nstigkeit ge�ngstigt, daher bestand die beste L�sung darin, sie am dunklen Portal zur�ckzulassen und ihnen L�ndereien zu versprechen. Auf diese Weise k�nnen sie die in der N�he des dunklen Portals verbliebenen Ork-Clans abfangen.;;;;;;;;;;;x
 EVTOPTA_WCBDP_67;Perfect!;;Hervorragend!;;;;;;;;;;;x
-EVTDESC_WCBDP_70;Orcs of the Laughing Skull clan have betrayed us! These bastards were laughing as they cut down our guards and fled through the Dark Portal to Azeroth! Although they helped us the first time, now these fools are on their own, preferring to raid and commit bloody rituals among the peaceful peoples of Azeroth.;;Die Orks aus dem Clan des lachenden Sch�dels haben uns verraten! Diese lachenden Bastarde metzelten unsere Wachen nieder und flohen durch das dunkle Portal nach Azeroth! Obwohl sie uns anfangs geholfen haben, sind diese Verr�ckten jetzt auf sich allein gestellt und ziehen Raubz�ge und blutige Rituale dem friedlichen Leben unter den V�lkern Azeroths vor.;;;;;;;;;;;x
+EVTDESC_WCBDP_70;The orcs of the Laughing Skull clan have betrayed us! These bastards were laughing as they cut down our guards and fled through the Dark Portal to Azeroth! Although they helped us the first time, now these fools are on their own, preferring to raid and commit bloody rituals among the peaceful peoples of Azeroth.;;Die Orks aus dem Clan des lachenden Sch�dels haben uns verraten! Diese lachenden Bastarde metzelten unsere Wachen nieder und flohen durch das dunkle Portal nach Azeroth! Obwohl sie uns anfangs geholfen haben, sind diese Verr�ckten jetzt auf sich allein gestellt und ziehen Raubz�ge und blutige Rituale dem friedlichen Leben unter den V�lkern Azeroths vor.;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCBDP_70;The orcs of the Laughing Skull clan have betrayed us! These bastards were laughing as they cut down our guards and fled through the Dark Portal to Azeroth! Although they helped us the first time, these fools are now loose on their own, preferring to raid and commit bloody rituals among the peaceful peoples of Azeroth.;;Die Orks aus dem Clan des lachenden Sch�dels haben uns verraten! Diese lachenden Bastarde metzelten unsere Wachen nieder und flohen durch das dunkle Portal nach Azeroth! Obwohl sie uns anfangs geholfen haben, sind diese Verr�ckten jetzt auf sich allein gestellt und ziehen Raubz�ge und blutige Rituale dem friedlichen Leben unter den V�lkern Azeroths vor.;;;;;;;;;;;x
```

Reply:
it is done, although I'm pretty sure, these have been changed only recently in the beyond the dark portal rework...